src/components/StoreInfo/StoreInfo.tsx

```diff
@@ -1,46 +1,41 @@
 import * as React from 'react';
 import styles from './styles.module.scss';
-import StoreDetails, { StoreDetailsProps } from '../StoreDetails';
+import StoreDetails from '../StoreDetails';
 import classnames from 'classnames';
+import { Address, Seller } from '../../utilities';
 
-interface Props {
-  nationality: string;
-  name: string;
-  address: {
-    line1: string;
-    line2: string;
-  };
-  phoneNumber: string;
-  introduction: string;
-  className?: string;
-  storeDetailsProps: StoreDetailsProps;
-}
+type Props = {
+  seller: Seller;
+};
+
+export const StoreInfo: React.SFC<Props> = ({ seller }) => {
+  const {
+    summary,
+    story,
+    className,
+    cuisineName,
+    addresses,
+    phoneNumber,
+  } = seller;
 
-const StoreInfo: React.SFC<Props> = ({
-  nationality,
-  name,
-  address,
-  phoneNumber,
-  introduction,
-  className,
-  storeDetailsProps,
-}) => {
-  const detailsProps = {
-    ...storeDetailsProps,
-    className: styles.storeDetails,
-  };
   return (
     <section className={classnames(styles.container, className)}>
-      <div className={styles.nationality}>{nationality}</div>
+      <div className={styles.nationality}>{cuisineName}</div>
       <p>
-        <div className={styles.address}>{address.line1}</div>
-        <div className={styles.address}>{address.line2}</div>
+        {addresses &&
+          addresses.map((address: Address) => (
+            <>
+              <div className={styles.address}>{address.address1}</div>
+              <div className={styles.address}>{address.address1}</div>
+              <div className={styles.address}>{address.city}</div>
+              <div className={styles.address}>{address.state}</div>
+              <div className={styles.address}>{address.zip_code}</div>
+            </>
+          ))}
         <div className={styles.address}>{phoneNumber}</div>
```

Comment:
Merp looking at this code made me realize something — I think that address should be renamed to 'locations". I'm also going to move this phone number to the address field. Sorry for the churn, but do you mind updating this? I'll update the backend in parallel